src/sign.builds

```diff
@@ -10,8 +10,26 @@
 
   <UsingTask AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll" TaskName="ReadSigningRequired" />
 
+  <PropertyGroup Condition="'$(OSGroup)' == 'Windows_NT' and '$(SkipSigning)' != 'true'">
+    <NativePlatform Condition="'$(Platform)' != 'AnyCPU'">$(Platform)</NativePlatform>
+    <NativePlatform Condition="'$(Platform)' == 'AnyCPU'">x64</NativePlatform>
+    <CLRCompressionLocation>$(BinDir)\$(OSGroup).$(NativePlatform).$(ConfigurationGroup)\native\clrcompression.dll</CLRCompressionLocation>
+    <AOTCLRCompressionLocation>$(BinDir)\$(OSGroup).$(NativePlatform).$(ConfigurationGroup)\native_aot\clrcompression.dll</AOTCLRCompressionLocation>
+  </PropertyGroup>
+  
+  <Target Name="GenerateSignForWindowsNative" 
+          Condition="'$(OSGroup)' == 'Windows_NT' and '$(SkipSigning)' != 'true' and Exists('$(CLRCompressionLocation)') and Exists('$(AOTCLRCompressionLocation)')"> 
+    <WriteSigningRequired AuthenticodeSig="$(AuthenticodeSig)"
```

Comment:
Strong Name signing doesn't apply to native binaries only managed binaries. I believe we want the StrongName value to be empty for these native binaries.